### PR TITLE
Feature/32-react-router

### PR DESCRIPTION
**Pull request by nhi-ngo**
-Add axios, react-router-dom dependencies
- Create new pages folder that includes Landing Page (path=/) and Crime Info page (path=/crime/state/{state-abbreviation}). Any other suggestion for URL path welcome!
- Click on the Logo to go back to Landing page.
- I tested and removed/added here and there where I see fit for now and make the app works :). Feel free to make any changes! I know it still needs a few changes like:
       * Remove the selected state when going back to the Landing Page
       * Configure how to make 'locate me' button to work with router
       * ......

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -39,6 +39,7 @@
     ]
   },
   "devDependencies": {
+    "axios": "^0.21.1",
```
Comment (theborgh) [inline package.json]: should be in dependencies (not devDependencies)

**Review by theborgh — APPROVED** [review]

**Review by theborgh — CHANGES_REQUESTED** [review]
please see comments above

**Comment by theborgh** [comment]
Looks good!

Just a quick note, axios should not be a devDependency (devDependencies do not get installed by the Heroku script when it's time to deploy the app. We need axios as a regular dependency because it's used in the actual app code to fetch the crime data. DevDependencies are for packages which are not needed in the actual production app (such as packages for testing, setting variable environments etc)

File: src/pages/CrimeInfo/CrimeInfo.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
+import React, { useState, useEffect } from "react";
+import axios from "axios";
+
+export default function CrimeInfo(props) {
+  const [info, setInfo] = useState([]);
+
+  const API_KEY = "e14y0OqlWGddCHCXLC3UzLBlcKecsGrxlahVvogx";
```
Comment (theborgh) [inline src/pages/CrimeInfo/CrimeInfo.js]: Better not to push api keys in the repository itself. We can achieve this with a devDependency called dotenv (we can discuss it later today if you want)

File: src/components/Navbar/SearchLocation/SearchLocation.js [hunk 3]
```diff
@@ -1,33 +1,26 @@
...
-        <option value="" hidden>
-          Select a state
-        </option>
-        {states.map(state => (
-          <option value={state.abbr} key={state.abbr}>
-            {state.name}
+  render() {
+    return (
+      <div>
+        <select select id="states" onChange={this.handleStateChange}>
```
Comment (theborgh) [inline src/components/Navbar/SearchLocation/SearchLocation.js]: I think this is just a typo but there should not be a "select" attribute here

**Review by cechae — APPROVED** [review]
I agree with Dario's points on the devdependency and api_key. It looks great otherwise!!

File: src/pages/CrimeInfo/CrimeInfo.js (hunk 2 above)
Comment (nhi-ngo) [reply src/pages/CrimeInfo/CrimeInfo.js]: Agree! I have looked into the dotenv package and was able to use it to hide the API.

**Comment by nhi-ngo** [comment]
Hey Dario and Jennifer, I made the changes based on you all recommendations. Thanks! I might have some git history and comments messed up while playing around with git rebase with this branch. But the final result should work as expected. Sorry for any confusion there.

File: src/components/Navbar/SearchLocation/SearchLocation.js (hunk 3 above)
Comment (nhi-ngo) [reply src/components/Navbar/SearchLocation/SearchLocation.js]: Thanks for spotting it. Just a typo!

**Review by theborgh — APPROVED** [review]

**Comment by theborgh** [comment]
> Hey Dario and Jennifer, I made the changes based on you all recommendations. Thanks! I might have some git history and comments messed up while playing around with git rebase with this branch. But the final result should work as expected. Sorry for any confusion there.

Everything looks good, thanks!

**Pull request merged**